<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicConf</class>
 <widget class="QDialog" name="GraphicConf">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphic parameters</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkInputs">
          <property name="text">
           <string>Border inputs</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkAAlias">
          <property name="text">
           <string>Antialiasing</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkStatus">
          <property name="text">
           <string>Status bar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkOGL">
          <property name="text">
           <string>OpenGL</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="checkLegend">
          <property name="text">
           <string>Legend</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Background color:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="ColorButton" name="colorBackground">
          <property name="useAlphaChannel">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Text color:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="ColorButton" name="colorText">
          <property name="useAlphaChannel">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string>Graphics</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="cbGraphicNames">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ColorButton" name="colorGraphic">
        <property name="useAlphaChannel">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Style:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboStyleGraphic"/>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkLines">
        <property name="text">
         <string>Lines width:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="spinLineWidthGraphic">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkPoints">
        <property name="text">
         <string>Points width:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="spinPointWidthGraphic">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkFill">
        <property name="text">
         <string>Fill:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="ColorButton" name="colorFill">
        <property name="useAlphaChannel">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Grid</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ColorButton" name="colorGrid">
        <property name="useAlphaChannel">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Style:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboStyleGrid"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="spinWidthGrid">
        <property name="decimals">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="spinGridStepX">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
        <property name="value">
         <double>50.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Step X:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Step Y:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="spinGridStepY">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
        <property name="value">
         <double>30.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="checkGridAutoX">
        <property name="text">
         <string>Auto X</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="checkGridAutoY">
        <property name="text">
         <string>Auto Y</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Margins</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="margin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="1" column="5">
       <widget class="QSpinBox" name="spinMarginR">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="spinMarginB">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelAll">
        <property name="text">
         <string>All:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinMarginL">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Right:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Left:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Bottom:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="spinMarginT">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Top:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinMarginT_2">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>colorbutton.h</header>
   <slots>
    <signal>colorChanged(QColor)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkAAlias</tabstop>
  <tabstop>checkOGL</tabstop>
  <tabstop>colorBackground</tabstop>
  <tabstop>colorText</tabstop>
  <tabstop>colorGrid</tabstop>
  <tabstop>comboStyleGrid</tabstop>
  <tabstop>spinWidthGrid</tabstop>
  <tabstop>cbGraphicNames</tabstop>
  <tabstop>colorGraphic</tabstop>
  <tabstop>comboStyleGraphic</tabstop>
  <tabstop>checkLines</tabstop>
  <tabstop>spinLineWidthGraphic</tabstop>
  <tabstop>checkPoints</tabstop>
  <tabstop>spinPointWidthGraphic</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GraphicConf</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>641</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkLines</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinLineWidthGraphic</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkPoints</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinPointWidthGraphic</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GraphicConf</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>641</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkFill</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorFill</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinMarginT_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinMarginT</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinMarginT_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinMarginR</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinMarginT_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinMarginB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinMarginT_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinMarginL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
